### PR TITLE
Added port & release to PR templates

### PR DESCRIPTION
**Pull request by jennuine**

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
Nice!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -19,7 +24,7 @@ before and after the PR.-->
 - [ ] All tests passed (See
   [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
```
Comment (adlarkin) [inline .github/PULL_REQUEST_TEMPLATE.md:25]:
```suggestion
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
```
I think making this text be all on the same line would fix the display issue I mentioned above :crossed_fingers:

**Review by adlarkin — COMMENTED** [review]
Looks pretty good to me! The one thing I have noticed is that the formatting is a little odd on some of the PRs I have seen using the current templates (a few things in the checklist are scattered across multiple lines). I've attached an image below to show an example (the last three items are scattered across multiple lines). I think that we can fix this by making all of the text for every checklist item be on the same line (I left a comment below that helps better illustrate an example). 

multiple lines template

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jennuine) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done, thanks! a62736ee5097160bf004f21d785c8ef46e3e4429

**Pull request merged**